backends/concrete-cuda/implementation/src/bootstrap_low_latency.cu

```diff
@@ -1,65 +1,107 @@
+#include "bootstrap_fast_low_latency.cuh"
 #include "bootstrap_low_latency.cuh"
-
 /*
  * Returns the buffer size for 64 bits executions
  */
 uint64_t get_buffer_size_bootstrap_low_latency_64(
     uint32_t glwe_dimension, uint32_t polynomial_size, uint32_t level_count,
     uint32_t input_lwe_ciphertext_count, uint32_t max_shared_memory) {
-  return get_buffer_size_bootstrap_low_latency<uint64_t>(
-      glwe_dimension, polynomial_size, level_count, input_lwe_ciphertext_count,
-      max_shared_memory);
-}
 
-// The number of samples should be lower than 4 * SM/((k + 1) * l) (the
-// factor 4 being related to the occupancy of 50%).
-bool verify_cuda_bootstrap_low_latency_grid_size_64(
-    int glwe_dimension, int polynomial_size, int level_count, int num_samples,
-    uint32_t max_shared_memory) {
-
-  int ret;
   switch (polynomial_size) {
   case 256:
-    ret = verify_cuda_bootstrap_low_latency_grid_size<uint64_t, Degree<256>>(
-        glwe_dimension, polynomial_size, level_count, num_samples,
-        max_shared_memory);
+    if (verify_cuda_bootstrap_fast_low_latency_grid_size<uint64_t, Degree<256>>(
```

Comment:
Here we should also check that the device supports cooperative groups (can be done later in a future PR, I'll open an issue).

Reply:
Here it is: https://github.com/zama-ai/concrete-core-internal/issues/533 (in the non urgent backlog)

Reply:
Yep, let's open an issue.